src/apis/HIBP.js

```diff
@@ -0,0 +1,26 @@
+const { APIWrapper } = require('../')
+const snekfetch = require('snekfetch')
+
+const API_URL = 'https://haveibeenpwned.com/api/v2'
+const USER_AGENT = 'Switchblade-Discord-Bot'
```

Comment:
no, it's not a key. it's called an User-Agent, and you need it so that HIBP can identify from where our requests come. that isn't going to interfere in nothing

Reply:
PS: marked as resolved, since there's nothing to fix.